pages/design/atomic-template-empty.md

```diff
@@ -50,7 +50,24 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- (e.g. WCAG Techniques or links with background information mentioned in Applicability, Expectations or Assumptions)
+### Understanding WCAG 
+
+- [WCAG 2.0 - Understanding 4.1.2: Name, Role, Value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)
```

Comment:
Can we use WCAG 2.1 links instead?

Reply:
... like this one: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html